Makefile

```
@@ -123,6 +130,50 @@ copr_build: srpm
 	@copr-cli --config $(_COPR_CONFIG) build $(_COPR_REPO) \
 		packaging/SRPMS/${PKGNAME}-${VERSION}-${RELEASE}*.src.rpm
 
+build: source
+	@echo "--- Build RPM ${PKGNAME}-${VERSION}-${RELEASE}.el$(DIST_VERSION).rpm ---"
+	@cp packaging/$(PKGNAME).spec packaging/$(PKGNAME).spec.bak
+	@sed -i "s/1%{?dist}/$(RELEASE)%{?dist}/g" packaging/$(PKGNAME).spec
+	@rpmbuild -ba packaging/$(PKGNAME).spec \
+		--define "_sourcedir `pwd`/packaging/sources"  \
+		--define "_srcrpmdir `pwd`/packaging/SRPMS" \
+		--define "_builddir `pwd`/packaging/BUILD" \
+		--define "_buildrootdir `pwd`/packaging/BUILDROOT" \
+		--define "_rpmdir `pwd`/packaging/RPMS" \
+		--define "rhel $(DIST_VERSION)" \
+		--define 'dist .el$(DIST_VERSION)' \
+		--define 'el$(DIST_VERSION) 1' || FAILED=1
+	@mv packaging/$(PKGNAME).spec.bak packaging/$(PKGNAME).spec
+
+build_container:
+	@case "$$BUILD_CONTAINER" in \
+	el7) \
+		_CONT_FILE="Containerfile.centos7"; \
+		;; \
+	el8) \
+		_CONT_FILE="Containerfile.ubi8"; \
+		;; \
+	f3[56]|rawhide) \
+		[ $$BUILD_CONTAINER = rawhide ] && VERSION=latest || VERSION=$${BUILD_CONTAINER: -2}; \
+		_CONT_FILE=".Containerfile.$${BUILD_CONTAINER}"; \
+		cp res/container-builds/Containerfile.fedora_generic res/container-builds/$$_CONT_FILE && \
+		sed -i "1i FROM fedora:$${VERSION}" res/container-builds/$$_CONT_FILE \
+		;; \
+	"") \
+		echo "BUILD_CONTAINER must be set"; \
+		exit 1; \
+		;; \
+	*) \
+		echo "Available containers are el7, el8, f35, f36, rawhide"; \
+		exit 1; \
+		;; \
+	esac && \
+	echo "--- Preparing $$BUILD_CONTAINER container for building ---" && \
+	IMAGE_NAME="leapp-build-$${BUILD_CONTAINER}"; \
+	podman build -f res/container-builds/$$_CONT_FILE -t $$IMAGE_NAME res/container-builds/ && \
+	podman run --rm -ti -v $$PWD:/payload:Z --name "$${IMAGE_NAME}-cont" $$IMAGE_NAME && \
+	[ $${_CONT_FILE:0:1} = '.' ] && rm -f res/container-builds/$$_CONT_FILE
```

@matejmatuska  If the condition result to false, the make commands ends with error and non-zero exit code which could be confusing.
```suggestion
	[ '$${_CONT_FILE:0:1}' = '.' ] && rm -f res/container-builds/$$_CONT_FILE || :
```

When fixed, we can merge it. Everything else seems to be working right.